airbyte-config/init/src/main/java/io/airbyte/config/init/LocalDefinitionsProvider.java

```diff
@@ -19,21 +19,25 @@
 import java.util.Map;
 import java.util.UUID;
 import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * This provider contains all definitions according to the local catalog json files.
  */
 final public class LocalDefinitionsProvider implements DefinitionsProvider {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(LocalDefinitionsProvider.class);
+
   private static final String LOCAL_CONNECTOR_CATALOG_PATH = CatalogDefinitionsConfig.getLocalConnectorCatalogPath();
 
   public CombinedConnectorCatalog getLocalDefinitionCatalog() {
     try {
       final URL url = Resources.getResource(LOCAL_CONNECTOR_CATALOG_PATH);
-      final String jsonString = Resources.toString(url, StandardCharsets.UTF_8);
-      final CombinedConnectorCatalog catalog = Jsons.deserialize(jsonString, CombinedConnectorCatalog.class);
-      return catalog;
+      LOGGER.info("Loading local catalog definitions from {}", url);
```

Comment:
nit: add the word `connector` in there

Reply:
also i'm not sure how often this will be logged - should probably only happen on initialization if possible